telosnetwork/teloscan
# #274 | Header overhaul

# Fixes #274

## Description
This PR introduces the new header. Implementation should generally match [the design](https://www.figma.com/file/MlvrttmWn1e27uITFsIrBg/v1.0.0?node-id=1%3A15&t=FVxnh0dgXeu9PRNy-0).

Note that the designs have Monitor and Endpoints as separate links, and though they are now a single page called Health since the design was created 

## Test scenarios
- go to https://deploy-preview-307--teloscan-stage.netlify.app/ on a **desktop** device
    - header should match designs
    - you should be able to `tab` through header elements
    - links should work
- click dark mode toggle 
    - dark mode should toggle as expected
- click 'sign in'
    - login modal should show
- log in
    - when logged in, header should match design; login status should be visible
    - address copy button and 'view address' button should work
- scroll down and up
    - scrolling down should hide the header
    - scrolling up should show header
    - let me know if you like/dislike this, it was not technically part of the acceptance criteria

Repeat for mobile (not including header scroll show/hide)

- log out 
- go to https://deploy-preview-307--teloscan-stage.netlify.app/staking
- click the button "Connect button"
    - login modal should pop

## Checklist:
